template.env

```
@@ -0,0 +1,11 @@
+# remove the template from this file (change it to '.env') and adjust the paths
+
+# absolute path to the RadonUlzer exe
+RadonUlzer_PATH=C:...\LineFollowerSim\program.exe
```

Maybe this would be clearer:
```suggestion
RadonUlzer_PATH=C:<Path to RadonUlzer>\LineFollowerSim\program.exe
```